TestApp/ios/Podfile

```diff
@@ -17,7 +18,7 @@ target 'TestApp' do
   pod 'React-RCTText', :path => '../node_modules/react-native/Libraries/Text'
   pod 'React-RCTVibration', :path => '../node_modules/react-native/Libraries/Vibration'
   pod 'React-RCTWebSocket', :path => '../node_modules/react-native/Libraries/WebSocket'
-
+  pod 'boost-for-react-native', :podspec => 'https://raw.githubusercontent.com/react-native-community/boost-for-react-native/master/boost-for-react-native.podspec'
```

Comment:
Why do we need that? Every new third party being introduced must be evaluated, especially after the up-to-date surge.

Reply:
It was here by mistake because I didn’t specify the main Cocoapods source 'https://github.com/CocoaPods/Specs.git’. Now we don’t need that, so I removed it.